tests/test_convert.py

```diff
@@ -1081,3 +1082,57 @@ def test_convert_hte_cellprofiler_csv(
 
     # clean up the parsl config for other tests
     parsl.clear()
+
+
+def test_cellhealth_preset(
```

Comment:
```suggestion
def test_legacy_cellprofiler_cytominerdatabase_cellheatlth_preset(
```

Reply:
or maybe just verbatim: `test_cell_health_cellprofiler_to_cytominer_database_legacy`

Reply:
Thanks for this, I'll go with the verbatim option and add this in.